<launch>

    <arg name="server" default="192.168.1.32"/>
    <arg name="v_id" default="2000"/>

  <!--  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" required="true">
            
        <rosparam subst_value="true">
            server: $(arg server)
            port: 3883

            update_frequency: 100.0
            frame_id: world

            # Use the VRPN server's time, or the client's ROS time.
            use_server_time: false
            broadcast_tf: true

            # Must either specify refresh frequency > 0.0, or a list of trackers to create
            refresh_tracker_frequency: 1.0
            #trackers:
            #- FirstTracker
            #- SecondTracker
        </rosparam>

        <remap 
            from="vrpn_client_node/V$(arg v_id)/pose" 
            to="V$(arg v_id)/mavros/mocap/pose" />

    </node>
-->

<node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>

    <remap 
            from="vrpn_client_node/V$(arg v_id)/pose" 
            to="V$(arg v_id)/mavros/mocap/pose" />
  </node>

</launch>